io.gitlab.gregorni.ASCIIImages.json

```diff
@@ -0,0 +1,86 @@
+{
+    "app-id" : "io.gitlab.gregorni.ASCIIImages",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "asciiimages",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.portal.OpenURI"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	{
+	    "name": "blueprint-compiler",
+	    "buildsystem": "meson",
+	    "sources": [
+	    	{
+		    "type": "git",
+		    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+		    "tag": "v0.6.0"
+	    	}
+	     ]
+	},
+	{
+            "name" : "jpeglib",
+            "builddir" : true,
+            "buildsystem" : "simple",
+            "build-commands" : [
+            	"./configure --prefix=/app/local",
+            	"make",
+            	"make install"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.ijg.org/files/jpegsrc.v9e.tar.gz",
+                    "sha256" : "4077d6a6a75aeb01884f708919d25934c93305e49f7e3f36db9129320e6f4f3d"
+                }
+            ]
+        },
+        {
+            "name" : "jp2a",
+            "builddir" : true,
+            "buildsystem" : "simple",
```

Comment:
same as above

Reply:
I need to pass "`--prefix=/app`" and "`--with-jpeg-prefix=/app`" to the configure script, though. How would I do that with `buildsystem : autotools`?

Reply:
that's what `config-opts` is for

Reply:
but jpeg isn't in `/app` since it's in the SDK.

Reply:
> but jpeg isn't in `/app` since it's in the SDK.

Oh, that might have been the problem.

Reply:
Hmm, removing the jpeg prefix end up with the same error. Maybe it has to do with passing the `-vi` flags to autoreconf?

Reply:
the autoreconf is run in a 'shell' source.

Reply:
What does that mean? Do I just add a `type : shell` source and use `commands : autoreconf -vi` in there?

Reply:
yes. Note that `commands` expect an array of strings, even if there is only one.

Reply:
Yeah, for example: https://github.com/flathub/org.gimp.GIMP/blob/d0c65eef8ed30c1fa8bbaf91309c0c8766ded724/org.gimp.GIMP.json#L561

Reply:
So not specifying the buildsystem actually seems to have fixed it. Thanks @barthalion , that was really helpful!

Reply:
that's because `autotools` is the default.

Reply:
Ah, I see.